nemo/collections/diffusion/models/flux/pipeline.py

```diff
@@ -685,12 +704,26 @@ def __init__(
         self.flux_controlnet = FluxControlNet(contorlnet_config) if flux_controlnet is None else flux_controlnet
 
     def load_from_pretrained(
-        self, flux_ckpt_path, controlnet_ckpt_path, do_convert_from_hf=True, save_converted_model_to=None
+        self,
+        flux_ckpt_path,
+        controlnet_ckpt_path,
+        do_convert_from_hf=True,
+        save_converted_model_to=None,
+        load_dist_ckpt=False,
     ):
         '''
         Converts both flux base model and flux controlnet ckpt into NeMo format.
         '''
-        if do_convert_from_hf:
+        assert not do_convert_from_hf and load_dist_ckpt, 'do_convert_from_hf and load_dist_ckpt cannot both be true'
```

Comment:
also here, remove the not:
        assert do_convert_from_hf and load_dist_ckpt, 'do_convert_from_hf and load_dist_ckpt cannot both be true'